Terminal.Gui/Configuration/AttributeJsonConverter.cs

```diff
@@ -37,7 +37,7 @@ public override Attribute Read (ref Utf8JsonReader reader, Type typeToConvert, J
         {
             if (reader.TokenType == JsonTokenType.EndObject)
             {
-                if ((foreground == null) || (background == null))
+                if (foreground == null || background == null)
```

Comment:
Null check should be ` if (foreground is not {} || background is not {} `

If ReSharper didn't do that, then either the cleanup profile that was run doesn't have that option checked or the style settings for c# don't have the null check pattern properly defined.